### DISCUSSION TITLE
URL opens two tabs

### DISCUSSION THREAD

**Discussion by josephj11** [keepassxreboot/keepassxc#6044] [Q&A] [answered]
I couldn't figure out the IRC, so I'm asking here.
I'm running KeepassXC 2.6.4 on kubuntu Linux. I used KeePassX for many years. Now I've been using KeepassXC for at least six months.

When I double click on a URL field, it opens two tabs in my browser - Vivaldi (based on chromium). This is relatively new, but it started in 2.6.3 or before. Until a month or two ago, it opened one tab as desired.
I wanted to ask about it here before filing an issue.

**Comment by phoerious** [comment, marked as answer]
This looks like there is something amiss with how mouse events are triggered. Can you reproduce it on another machine? Because I cannot and it doesn't really make sense to me.

**Reply by josephj11** [reply]
I'll try, but it may take a few days before I can test it.
You guys really respond quickly! :)

**Reply by josephj11** [reply]
This is weirder than I thought. The good news is that it isn't your problem. The bad news is that icons on my desktop are doing it too - which is what proves it isn't your problem.
